tensorflow_encrypted/protocol/pond.py

```diff
@@ -898,6 +898,9 @@ def eval(self, sess, feed_dict={}, tag=None) -> np.ndarray:
         value = self.value_on_0.eval(sess, feed_dict=feed_dict, tag=tag)
         return self.prot._decode(value, self.is_scaled)
 
+    def __getitem__(self, slice: slice) -> 'PondPublicTensor':
+        return PondPublicTensor(self, self.value_on_0[slice], self.value_on_1[slice], is_scaled=self.is_scaled)
```

Comment:
same here, even if no security issue, to be safe on performance it's worth pinning to device here